library/psa_crypto_cipher.h

```diff
@@ -24,22 +24,6 @@
 #include <mbedtls/cipher.h>
 #include <psa/crypto.h>
 
-/** Get Mbed TLS cipher information given the cipher algorithm PSA identifier
```

Comment:
This declaration belongs here. Please remove the copy in `psa_crypto_core.h` instead.

Reply:
It looked to me like it wasn't a good fit for this file - the docs at the top of psa_crypto_cipher.h say "PSA cipher driver entry points", and this isn't a driver entry point? Happy to switch it round if preferred though.

Reply:
The function is defined in `psa_crypto_cipher.c`. The description should probably have “and associated auxiliary functions” or some such.